src/processevent.cpp

```diff
@@ -18,7 +18,8 @@ namespace ProcessEvent
         QJsonDocument jsonDoc;
         if (!message.startsWith("{"))
         {
-            auto str = QString("{\"idx\": %1, \"k\": \"\", \"v\": \"%2\"}").arg(QString::number(index), message);
+            auto str = QString("{\"idx\": %1, \"k\": \"\", \"v\": \"%2\"}")
+                    .arg(QString::number(index), message.replace("\\", "\\\\"));
```

Comment:
We have to replace/escape the "\" char, and I think replace() is as good as we get. Opening a 200 MB file goes from 38 secs to 44 secs. The increase is well worth the correct display of text...